src/server/pfs/cmds/cmds.go

```diff
@@ -463,16 +463,24 @@ Files and URLs should be newline delimited.
 					if err != nil {
 						return err
 					}
+					defer func() {
```

Comment:
Should we add a note about this (and possibly the filepath := scanner.Text() part below) to the PR description?